src/phoenix/db/models.py

```diff
@@ -619,65 +618,84 @@ class ExperimentRunAnnotation(Base):
-        name: Mapped[str]
-        description: Mapped[Optional[str]]
-        created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
-        expires_at: Mapped[Optional[datetime]] = mapped_column(UtcTimeStamp)
-
-    # todo: standardize audit table format (https://github.com/Arize-ai/phoenix/issues/4185)
-    class AuditAPIKey(Base):
-        __tablename__ = "audit_api_keys"
-        id: Mapped[int] = mapped_column(primary_key=True)
-        api_key_id: Mapped[int] = mapped_column(
-            ForeignKey("api_keys.id"),
-            nullable=False,
-            index=True,
-        )
-        user_id: Mapped[int] = mapped_column(
-            ForeignKey("users.id"),
-            nullable=False,
-            index=True,
-        )
-        action: Mapped[str] = mapped_column(
-            CheckConstraint("action IN ('CREATE', 'DELETE')", name="valid_action")
-        )
-        created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
+class User(Base):
+    __tablename__ = "users"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    user_role_id: Mapped[int] = mapped_column(
+        ForeignKey("user_roles.id"),
+        index=True,
+    )
+    role: Mapped["UserRole"] = relationship("UserRole", back_populates="users")
+    username: Mapped[Optional[str]] = mapped_column(nullable=True, unique=True, index=True)
+    email: Mapped[str] = mapped_column(nullable=False, unique=True, index=True)
+    auth_method: Mapped[str] = mapped_column(
+        CheckConstraint("auth_method IN ('LOCAL')", name="valid_auth_method")
+    )
+    password_hash: Mapped[Optional[str]]
+    reset_password: Mapped[bool]
+    created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
+    updated_at: Mapped[datetime] = mapped_column(
+        UtcTimeStamp, server_default=func.now(), onupdate=func.now()
+    )
+    deleted_at: Mapped[Optional[datetime]] = mapped_column(UtcTimeStamp)
+    access_tokens: Mapped[List["AccessToken"]] = relationship("AccessToken", back_populates="user")
+    refresh_tokens: Mapped[List["AccessToken"]] = relationship(
+        "RefreshToken", back_populates="user"
+    )
+    api_keys: Mapped[List["ApiKey"]] = relationship("ApiKey", back_populates="user")
+
+
+class AccessToken(Base):
+    __tablename__ = "access_tokens"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    user_id: Mapped[int] = mapped_column(ForeignKey("users.id"), index=True)
+    user: Mapped["User"] = relationship("User", back_populates="access_tokens")
+    created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
+    expires_at: Mapped[Optional[datetime]] = mapped_column(UtcTimeStamp, index=True)
+
+
+class RefreshToken(Base):
+    __tablename__ = "refresh_tokens"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    user_id: Mapped[int] = mapped_column(ForeignKey("users.id"), index=True)
```

Comment:
Note @axiomofjoy talked about associating a refresh token to an access token

Reply:
ok sg. this is just copy-pasta from the access token table above

Reply:
Would make sense to add a foreign key relation like Mikyo mentioned.

Reply:
added